### DISCUSSION TITLE
Add one more option/role in project access as can edit content

### DISCUSSION THREAD

**Discussion by beingminimal** [community/community#67821] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

In github project we need to lock views for other team members to make this workflow more relaibale for teams.
So in case new member join the team then they will not meshup project views which already took time to setup for different workflows.

So we can learn from notion sharing management and can adopt for github project roles also.
We need to add one more role for github project named as shown in below image which is
**"Can edit content"**
Can edit issues, PR & discussions but can't change project filters and views and structure.

![Screenshot 2023-09-21 17 52 58](https://github.com/community/community/assets/84431724/0dd9d84c-698e-42e5-81d7-31192be52590)

So project managers can have more control over their workflows and teams can work seamlessly. 
Thank You.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by beingminimal** [comment]
@ankneis any consideration for this feature?

**Comment by beingminimal** [comment]
@queenofcorgis @azenMatt @evi-liu can we have this features soon?

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
